packages/pwa/package.json

```diff
@@ -113,16 +109,12 @@
         "eslint-plugin-jsx-a11y": "6.4.1",
         "eslint-plugin-prettier": "3.0.1",
         "eslint-plugin-react": "7.24.0",
-        "execa": "^1.0.0",
         "full-icu": "^1.3.1",
-        "glob": "7.1.2",
         "html-loader": "0.5.5",
         "ignore-loader": "^0.1.2",
         "ignore-styles": "5.0.1",
-        "ip": "1.1.5",
         "jest": "^26.6.3",
         "jest-cli": "^26.6.3",
-        "minimist": "1.2.5",
         "msw": "^0.28.1",
         "njwt": "^1.0.0",
         "nock": "^13.1.1",
```

Comment:
I don't see `nock` being used in the PWA, but I see that it's defined as a comment dependency in said script. Is there something we can do here.. potentially not make it a common dep? I'm not 100% familiar with that script so we'd want to be cautious if we make changes there.

Reply:
Yup, I'm not familiar with the script either. I see that `nock` is only used within the sdk package and nowhere else. So it seems like we can remove it for all packages except the SDK. 

But... again, I'm not familiar with the context of the script. So I've created a [ticket](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000i1IPtYAM/view) for us to look at this in the future.